BO4E/BO/Entsperrauftrag.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+using BO4E.ENUM;
+using Newtonsoft.Json;
+
+namespace BO4E.BO
+{
+    /// <summary>
+    /// Der Entsperrauftrag ist sehr ähnlich aufgebaut wie der <see cref="Sperrauftrag"/>.
+    /// </summary>
+    /// <remarks>Er wird typischerweise mit einer EDIFACT-Nachricht des Typs 17117 kommuniziert.</remarks>
+    public class Entsperrauftrag : Auftrag
+    {
+        /// <summary>
+        /// <inheritdoc cref="Sperrauftrag.Sperrauftragsart"/>
+        /// </summary>
+        [JsonProperty("sperrauftragsart", Required = Required.Always)]
+        [JsonPropertyName("sperrauftragsart")]
+        public Sperrauftragsart Sperrauftragsart => Sperrauftragsart.ENTSPERREN;
+
+        /// <summary>
+        /// <see cref="ENUM.Sperrauftragsstatus"/>
+        /// </summary>
+        [JsonProperty("sperrauftragsstatus", Required = Required.Default)]
+        [JsonPropertyName("sperrauftragsstatus")]
+        public Sperrauftragsstatus? Sperrauftragsstatus { get; set; }
+
+        /// <summary>
+        /// Ist true, falls die Entsperrung innerhalb der Arbeitszeit zu erfolgen hat.
+        /// </summary>
+        /// <remarks>Falls true ist in EDIFACT Z53 zu verwenden, falls false dann Z54</remarks>
+        [JsonProperty("istNurInnerhalbDerArbeitszeitZuEntsperren", Required = Required.Default)]
```

Comment:
Wenn ich es richtig sehe, muss man es bei einer Entsperrung immer angeben oder?
```suggestion
        [JsonProperty("istNurInnerhalbDerArbeitszeitZuEntsperren", Required = Required.Always)]
```